Document GitLab access token (#126)
CloudCannon/platform-documentation: articles/connecting-a-gitlab-respository-as-your-source.mdx

<comp.DocsImage path="ye_olde_images/files-source-syncing-gitlab.png" alt="Selecting GitLab as your source provider" type="screenshot" />

This will open a menu where you can connect to GitLab, via one of two methods.

## Connecting with your user account

You can connect to GitLab by authenticating with your user account. This will give your CloudCannon organization access to the same GitLab repositories you have access to.

> This option will redirect you to GitLab. Log in and authorize CloudCannon access to your GitLab account. You'll be redirected back to CloudCannon to pick a repository to connect.

Reviewer: ```suggestion
This option will redirect you to GitLab. Log in and authorize CloudCannon access to your GitLab account, after which you will be redirected back to CloudCannon to pick a repository to connect.
```

<comp.DocsImage path="ye_olde_images/files-source-syncing-gitlab-oauth.png" alt="A menu with a button labelled 'Authenticate with GitLab'." type="screenshot" />

## Authenticating with a group access token

> Alternatively, you can create a group access token in GitLab and save that token in CloudCannon. This will give your CloudCannon organization access to all the resources scoped to your token.

Reviewer: ```suggestion
Alternatively, you can create a group access token in GitLab and save that token in CloudCannon. This will give your CloudCannon organization access to all the resources scoped to your token.

See GitLab's [Group access tokens](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html) documentation for how to create and configure a token.

<comp.Notice info_type="important">
  For full functionality on CloudCannon, your group access token should have (at least) the Maintainer role, and the `api`, `read_api`, `read_repository`, and `write_repository` scopes.
</comp.Notice>
```

Copy the token into the **Group Access Token** field. You will also need to provide the expiry date of the token. CloudCannon will send a warning email to all owners of your organization when the token is about to expire.

> <comp.Notice info_type="important">
>   For full functionality on CloudCannon, your group access token should have (at least) the Maintainer role, and the `api`, `read_api`, `read_repository`, and `write_repository` scopes.
> </comp.Notice>

Reviewer: moved above
```suggestion
```
